### ISSUE TITLE
Scaling problem for 4k displays.

### ISSUE THREAD

**Issue by Snellingen** [bradleyasu/ytdl50#7]
This program does not scale correctly and makes the text very small on a high dpi monitor.

**Comment by bradleyasu** [comment]
This is true - The application is written in in Java Swing which doesn't officially support/scale correctly for 4k displays.  I have been looking into some alternatives though, I'm leaving this issue open until I figure out a solution... I have a 4k display as well and this is very annoying.
